lib/options.nix

```diff
@@ -147,6 +147,17 @@ rec {
       mkLuaFn' = args: mkNullOrLuaFn' (processDefaultNullArgs args);
       mkLuaFn = pluginDefault: description: mkLuaFn' { inherit pluginDefault description; };
 
+      mkRaw' =
+        args:
+        mkNullable' (
+          args
+          // {
+            type = types.rawLua;
+            pluginDefault = lib.nixvim.mkRaw args.pluginDefault;
```

Comment:
`args.pluginDefault` should be optional. We'll need something like:

```nix
mkRaw' =
  args:
  mkNullable' (
    args
    // { type = types.rawLua; }
    // lib.optionalAttrs (args ? pluginDefault) {
      pluginDefault = lib.nixvim.mkRaw args.pluginDefault;
    }
  );
```

Reply:
Good catch, done.